pages/1.10/installing/evaluation/gcp/index.md

```diff
@@ -99,23 +99,38 @@ Password: `deleteme`
   - ```public-agent-loadbalancer``` - Specifies the URL of your Public routable services.
 
   ```hcl
+  provider "google" {
+    version = "~> 1.0"
```

Comment:
think we should suggest `~> 1.18.0` to avoid confusion with the depreciation notice from `1.19` on